README.md

```diff
@@ -1,6 +1,16 @@
-# My Dockerfiles
+# Akeneo PIM server with Docker
 
-This repository contains the Dockerfiles I use for Akeneo and other Symfony development. Feel free to use/adapt them if they fit your needs.
+Akeneo stack is also available as a Docker image, provided and maintained by Akeneo.
```

Comment:
Add a mention that these images do not contains Akeneo PIM itself, only a running environment for test and dev purpose.